04_12_2019/development/main.cpp

```diff
@@ -5,6 +5,40 @@
 int main()
 {
 
-	Matrix m;
+	Matrix m(3,3);
+	m[0][1] = 2;
+	m[1][0] = 5;
+	m[1][2] = 7.5;
+	m[2][2] = 10;
+	// output initial matrix
+
+	std::cout << "original matrix:" << std::endl;
+	
+	for (int i = 0; i < m.numberOfRows(); ++i)
+	{
+		for (int j = 0; j < m.numberOfCols(); ++j)
+		{
+			std::cout << m(i, j) << "\t";
+		}
+		std::cout << std::endl;
+	}
+
+	std::cout << std::endl;
```

Comment:
There is the overloading for `<<` operator.  `std::cout<<m;` should be more than enough. Same for the second one.